apps/search/src/app/app.module.ts

```diff
@@ -11,6 +11,7 @@ import { storeFreeze } from 'ngrx-store-freeze'
 import { environment } from '../environments/environment'
 import { AppRoutingModule } from './app-routing.module'
 import { AppComponent } from './app.component'
+import { LibCatalogModule } from '../../../../libs/catalog/src/lib/catalog.module'
```

Comment:
you need to add an alias path in root tsconfig, links between libraries should be aliased, not relative.

Reply:
Can't this be manage by CLI? I probably did something wrong when creating `catalog` in lib? 

Added to `tsconfig.base.json`. No change.

Then got
```
Error: /data/dev/gn/ui/libs/catalog/src/index.ts is missing from the TypeScript compilation. Please make sure it is in your tsconfig via the 'files' or 'include' property.
```
Added `catalog/src/index.ts` was missing.

After a build it looks ok now but in Intellij
![image](https://user-images.githubusercontent.com/1701393/96329026-52a2f380-1049-11eb-9791-83928c121377.png)